properties-files/openshiftv4_on_gcp.properties

```diff
@@ -0,0 +1,65 @@
+ORCHESTRATION_USER=root
+ORCHESTRATION_HOST=
+SSHKEY_TOKEN=
+OPENSHIFT_CLEANUP=false
+OPENSHIFT_INSTALL=true
+OPENSHIFT_POST_INSTALL=true
+OPENSHIFT_POST_CONFIG=true
+OPENSHIFT_DEBUG_CONFIG=false
+OPENSHIFT_CLIENT_LOCATION=
+SCALE_CI_BUILD_TRIGGER=true
+SCALE_CI_BUILD_TRIGGER_URL=
+OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=
+OPENSHIFT_INSTALL_BINARY_URL=
+ENABLE_DITTYBOPPER=true
+ENABLE_REMOTE_WRITE=false
+SINCGARS_REMOTE_WRITE_URL=
+OPENSHIFT_INSTALL_APIVERSION=v1
+OPENSHIFT_INSTALL_SSH_PUB_KEY_FILE=/root/.ssh/id_rsa.pub
+OPENSHIFT_INSTALL_PULL_SECRET=
+OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_IMAGE_REGISTRY=
+OPENSHIFT_INSTALL_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE=false
+OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE_VERSION=
+GOPATH=/root/.go
+GCP_SERVICE_ACCOUNT=
+GCP_SERVICE_ACCOUNT_EMAIL=
+GCP_AUTH_KEY_FILE=
+GCP_PROJECT=
+GCP_REGION=
+OPENSHIFT_BASE_DOMAIN=
+OPENSHIFT_CLUSTER_NAME=
+OPENSHIFT_MASTER_COUNT=3
+OPENSHIFT_WORKER_COUNT=5
+OPENSHIFT_MASTER_INSTANCE_TYPE=n1-standard-4
+OPENSHIFT_WORKER_INSTANCE_TYPE=n1-standard-4
+OPENSHIFT_MASTER_ROOT_VOLUME_SIZE=64
```

Comment:
Let's set the defaults to match the installer. Relevant issue: https://github.com/openshift-scale/scale-ci-deploy/issues/72.